examples/federated-learning/tff/distributed-fl-simulation-k8s/README.md

```diff
@@ -156,10 +158,38 @@ You can run this example in different runtime environments:
      that exposes the second worker workloads.
 
 1. Run `terraform apply`.
+1. [Build the example container image, and push it to the container image registry](#build-the-example-container-image-and-push-it-to-the-container-image-registry).
 1. Wait for GKE to report the coordinator and the workers as `Ready` in the
    [GKE Workloads dashboard](https://cloud.google.com/kubernetes-engine/docs/concepts/dashboards#workloads)
    in their respective GKE clusters.
 
+### Build the example container image, and push it to the container image registry
+
+1. Build the example container image locally on your host:
+
+   ```sh
+   DISTRIBUTED_TFF_EXAMPLE_CONTAINER_IMAGE_REPOSITORY_HOSTNAME="$(terraform output -raw container_image_repository_fully_qualified_hostname)"
+   DISTRIBUTED_TFF_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID="${DISTRIBUTED_TFF_EXAMPLE_CONTAINER_IMAGE_REPOSITORY_HOSTNAME}/$(terraform output -raw container_image_repository_name)/tff-runtime:latest"
```

Comment:
Do you think it's a good idea to tag this container as latest ? I'll tag a fixed version to have reproducibility

Reply:
We can tag it what whatever, but for an example it's probably fine. For reproducibility, you'd want to use a hash maybe.

I can use something like `0.0.1` or whatever, if you prefer :)

Reply:
Let's use `0.0.1` for reproducibility purpose :)